internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -533,6 +533,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 
 	// Note that this 'Composition' will be derived from a
 	// CompositionRevision if the relevant feature flag is enabled.
```

Comment:
This comment is stale - revisions are GA and can't be disabled. Not related to this PR but it would be nice to update if doing so is easy.

Reply:
done